lib/extensions/ar_migration.rb

```diff
@@ -1,6 +1,17 @@
 module ArPglogicalMigrationHelper
+  SCHEMA_MIGRATIONS_RAN_MIGRATION = "20171031010000".freeze
+
+  def self.subscriptions
+    @subscriptions ||= PglogicalSubscription.all
+  end
+
+  def self.schema_migrations_ran_exists?
+    ActiveRecord::Base.connection.table_exists?("schema_migrations_ran")
```

Comment:
This doesn't seem to be very expensive, maybe because it's a schema check?  It doesn't seem to make a difference and in fact checking if version <=> SCHEMA_MIGRATIONS_RAN_MIGRATION is more costly somehow.